vllm/model_executor/models/minicpmv.py

```diff
@@ -379,6 +380,10 @@ def input_mapper_for_minicpmv(ctx: InputContext, data: object):
     if image_processor is None:
         raise RuntimeError("No HuggingFace processor is available "
                            "to process the image object")
+
+    if not isinstance(data, list) or len(data) <= 0:
+        raise ValueError("Invalid image input (%s)", data)
```

Comment:
Users are currently allowed to pass an empty list of images. We should still allow this case.